maven-embedder/src/main/mdo/core-extensions.mdo

```diff
@@ -82,6 +82,14 @@
           <required>true</required>
           <type>String</type>
         </field>
+        <field>
+          <name>classloadingStrategy</name>
```

Comment:
I think this should be `classLoadingStrategy` because the object is called `ClassLoader`